src/util/clickTracking.test.js

```diff
@@ -0,0 +1,66 @@
+import * as clickTracking from './clickTracking';
+
+/*
+ * **** DISABLED ******
+ *
+ * This is disabled because tests don't have `global.documnet` defined in
```

Comment:
It is strange, and I'm not sure why it behaves so differently in Travis, but it seemed like a bigger issue than what I could tackle in this branch. Even locally, it was very awkward to try to get the `window.Event` object to reset correctly after each test. This script would ideally get tested in a real browser environment but I can file a followup ticket to see whether it can be refactored to be a little more node-friendly unless you feel that it's a blocker.